src/swift/flwr/Sources/Flower/GRPC/FlwrGRPC.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  File.swift
```

Comment:
copy/paste?

Reply:
When creating a new file, File.swift is automatically created. Will be changed accordingly.

Reply:
Thank you.